derive_builder/src/lib.rs

```diff
@@ -367,6 +367,28 @@
 //! }
 //! ```
 //!
+//! ## Additional Trait Derivations
+//!
+//! You can derive additional traits on the builder, including traits defined by other crates:
+//!
+//! ```rust
+//! # #[macro_use]
+//! # extern crate derive_builder;
+//! #
+//! #[derive(Builder, Clone)]
+//! #[builder(derive(Debug, PartialEq, Eq))]
+//! pub struct Lorem {
+//!     foo: u8,
+//!     bar: String,
+//! }
+//! 
+//! fn main() {
+//!    assert_eq!(LoremBuilder::default(), LoremBuilder::default());
+//! }
+//! ```
+//!
+//! Attributes declared for those crates are _not_ forwarded to the fields on the builder.
```

Comment:
That should be "traits", not "crates" - I'll fix that. I agree with unconditional derivation for the `Debug` and `Clone` traits.